src/commands/publish.ts

```diff
@@ -72,6 +72,17 @@ export const builder: CommandBuilder = (yargs: Argv) => {
         'Source revision (git SHA or tag) to publish (if not release branch head)',
       type: 'string',
     })
+    .option('merge-target', {
```

Comment:
I've often seen "base branch" used as terminology for this, but "merge target" is equally clear.